src/main/scala/com/bwsw/cloudstack/vault/server/cloudstack/util/ApacheCloudStackTaskWrapper.scala

```diff
@@ -0,0 +1,79 @@
+package com.bwsw.cloudstack.vault.server.cloudstack.util
+
+import java.util.UUID
+
+import br.com.autonomiccs.apacheCloudStack.client.{ApacheCloudStackClient, ApacheCloudStackRequest}
+import br.com.autonomiccs.apacheCloudStack.client.beans.ApacheCloudStackUser
+import com.bwsw.cloudstack.vault.server.cloudstack.entities.Tag
+import com.bwsw.cloudstack.vault.server.util.{ApplicationConfig, ConfigLiterals}
+import org.slf4j.LoggerFactory
+
+import scala.util.{Failure, Success, Try}
+
+/**
+  * Created by medvedev_vv on 21.08.17.
+  */
+class ApacheCloudStackTaskWrapper {
```

Comment:
ApacheCloudStackTaskWrapper  -> ApacheCloudStackTaskCreator
add to each method the following prefix: "create"